pkg/controllers/prometheus-operator/controller.go

```diff
@@ -80,8 +82,8 @@ func (r *reconciler) Reconcile(ctx context.Context, request ctrlruntime.Request)
 	return ctrlruntime.Result{}, nil
 }
 
-func (r *reconciler) loadStaticResources() ([]client.Object, error) {
-	resources := []assets.Asset{
+func (r *reconciler) loadStaticResources(namespace string) ([]client.Object, error) {
```

Comment:
nit: isn't the namespace already accessible through `r.opts` ?

Reply:
Correct, removed.